jina/helloworld/multimodal/flow-index.yml

```diff
@@ -7,19 +7,19 @@ executors:
       metas:
         workspace: $HW_WORKDIR
         py_modules:
-          - my_executors.py
+          - $APP_DIR/my_executors.py
```

Comment:
This is done to solve the issue with yaml register_cls / pymodule imports in a spawned process. Should be reverted